### DISCUSSION TITLE
关于脚本被greasyfork网站删除后violentmonkey更新的问题

### DISCUSSION THREAD

**Discussion by violenttamper** [violentmonkey/violentmonkey#1272] [Q&A] [answered]
我想知道，如果某个在线脚本被greasyfork或其他脚本网站删除了，violentmonkey如果可以更新，更新后会使得脚本不可用吗？

**Comment by tophf** [comment, marked as answer]
It can't be updated but it will be usable as long as you disable updates for this script.

**Reply by violenttamper** [reply]
请问，如何禁用更新？

**Reply by tophf** [reply]
![image](https://user-images.githubusercontent.com/1310400/117136100-93c5df80-adb0-11eb-99f9-a1ed53df0def.png)

**Reply by violenttamper** [reply]
感谢，解答
